docs/docs/tracing/sdk/manual-instrumentation.mdx

```diff
@@ -0,0 +1,238 @@
+---
+sidebar_position: 1
+sidebar_label: Manual Tracing
+---
+
+import { APILink } from "@site/src/components/APILink";
+import { Card, CardGroup, SmallLogoCard } from "@site/src/components/Card";
+import TOCInline from "@theme/TOCInline";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Manual Tracing
+
+In addition to the [Auto Tracing](..#automatic-tracing) integrations, you can instrument your Python code using the MLflow Tracing SDK. This is especially useful when you need to instrument your custom Python code.
+
+## Decorator
```

Comment:
note: Intentionally avoid the term "fluent API" as we got feedbacks that the concept is hard to understand. I haven't seen a user who actually use client APIs, so here we just show it as a standard APIs and hide the client APIs at the bottom.

Reply:
We should probably consider removal of that term elsewhere too. Even with the tracking APIs, the vast majority of users utilize those fluent APIs with the context manager for run initiation. Makes sense to be consistent with this information presentation.

Reply:
Agreed, did a quick code search but luckily we don't use the term too frequently. We can do a follow-up to clean them up entirely:)